src/pipecat/processors/audio/audio_buffer_processor.py

```diff
@@ -22,8 +22,7 @@
 
 class AudioBufferProcessor(FrameProcessor):
     """This processor buffers audio raw frames (input and output). The mixed
-    audio can be obtained by calling `get_audio()` (if `buffer_size` is 0) or by
-    registering an "on_audio_data" event handler. The event handler will be
+    audio can be obtained by registering an "on_audio_data" event handler. The event handler will be
```

Comment:
Good catch. Thank you! Do you mind wrapping the line so it doesn't stick out form the other lines?